From 3e9d1c6b7dabb933c1ab18d14ddd4eb704901a3b Mon Sep 17 00:00:00 2001
From: Manus <manus@example.com>
Date: Sun, 1 Feb 2026 06:34:06 -0500
Subject: [PATCH] Add node_modules to .gitignore

---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 .gitignore

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 00000000..e2125946
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1,2 @@
+
+node_modules
-- 
2.34.1

